go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/gorilla/mux v1.7.3
 	github.com/grpc-ecosystem/grpc-gateway v1.13.0 // indirect
 	github.com/spf13/cobra v0.0.6
```

Comment:
This seems to be pulling in changes from https://github.com/tektoncd/triggers/pull/684

a `git pull --rebase upstream master` might help